### ISSUE TITLE
Breaking the Docker build (again)

### ISSUE THREAD

**Issue by keirf** [planeturban/docker-flashfloppy#1]
Hi there, I'm making changes to FlashFloppy recommended build env: Ubuntu 20.04 and using Python3. You can see the prereq steps in FlashFloppy's new CI workflow file and I updated the wiki instructions too. It would be great if Dockerfile could be updated to match.

EDIT: https://github.com/keirf/FlashFloppy/blob/master/.github/workflows/ci.yml

EDIT2: Basically something like below:
```
sudo apt -y install git gcc-arm-none-eabi python3-pip srecord stm32flash zip unzip wget
python3 -m pip install --user crcmod intelhex
```

**Comment by planeturban** [comment]
Thanks! I'm on it! Forgot to lock the ubuntu version so technically I broke it myself as well. :D

**Comment by planeturban** [comment]
Fixed in eaf95c9, locked to 20.04 and correct deb packages installed.
